app/src/test/java/org/fossasia/openevent/app/presenter/AttendeePresenterTest.java

```diff
@@ -122,6 +122,8 @@ public void shouldLoadAttendeesSuccessfully() {
 
         attendeesPresenter.loadAttendees(false);
 
+        Collections.sort(attendees);
```

Comment:
but in 
```
inOrder.verify(attendeesView).showAttendees(attendees);
```
`attendees` is not from the presenter, it is from the test context, which is not sorted